lib/overcommit/hook/pre_commit/puppet_metadata_json_lint.rb

```diff
@@ -0,0 +1,32 @@
+module Overcommit::Hook::PreCommit
+  #
+  # Run's the Puppet metadata linter. It has support for adding options
+  # in the .overcommit.yaml
+  #
+  # PreCommit:
+  #   PuppetMetadataJsonLint:
+  #     enabled: true
+  #    flags: ['--no-strict_license', '--no-strict-dependencies', '--fail-on-warning']
```

Comment:
Indentation is a bit off. Also underscore should probably be hyphen.

Taking a step back, I don't see a strong need to include this example YAML as a comment since it's just duplicated from `config/default.yml`.